File: Client/Assets/iTweenExamples/WorkingWithPhysics/Scripts/Brick.js
private var stableColor : Color = Color.white;
private var impactColor : Color = Color.red;

function Update (){
	if(GetComponent.<Rigidbody>().velocity.magnitude>2){
		iTween.ColorTo(gameObject,impactColor,.3);
	}else{
		iTween.ColorTo(gameObject,stableColor,.3);
	}
}

File: Client/Assets/iTweenExamples/WorkingWithPhysics/Scripts/Brick.cs
using UnityEngine;
using System.Collections;

public class Brick : MonoBehaviour
{
	private Color stableColor = Color.white;
	private Color impactColor = Color.red;
	
	void Update (){
		if(GetComponent<Rigidbody>().velocity.magnitude>2){
			iTween.ColorTo(gameObject,impactColor,.3f);
		}else{
			iTween.ColorTo(gameObject,stableColor,.3f);
		}
	}
}

